src/Components/InfoBox/index.jsx

```
@@ -0,0 +1,42 @@
+import { RowContainer } from "../StandardContainer";
+import { Stack, Typography } from "@mui/material";
+import Image from "../Image";
+
+import { useTheme } from "@mui/material/styles";
+
+const InfoBox = ({
+	img,
```

ESLint

[error] 'img' is missing in props validation

(react/prop-types)